File: Modulo17/Pessoas/Pessoa.cpp
#include "Pessoa.h"
#include <iostream>

using namespace std;

Pessoa::Pessoa() {
}

Pessoa::Pessoa(string nome, int idade) : nome(nome), idade(idade) {
}

Pessoa::~Pessoa() {
}

void Pessoa::imprime() const {
    cout << "Pessoa : " << nome << ", " << idade << endl;
}

bool Pessoa::compara(const Pessoa p1, const Pessoa p2) {
    if (p1.idade == p2.idade) {
        return p1.nome < p2.nome;
    } else {
        return p1.idade < p2.idade;
    }
}

File: Modulo05/Package/br/ufscar/dc/package1/Pessoa.java
package br.ufscar.dc.package1;

import javax.swing.plaf.basic.BasicInternalFrameTitlePane.SystemMenuBar;

public class Pessoa {

    private String nome;
    private int idade;

    public Pessoa(String nome, int idade) {
        this.nome = nome;
        this.idade = idade;
    }

    public void imprime() {
        System.out.println("Nome: " + nome);
        System.out.println("Idade: " + idade);
    }
}